modules/twinkleprod.js

```diff
@@ -386,7 +386,7 @@ Twinkle.prod.callbacks = {
 		if (params.logEndorsing) {
 			logText += 'endorsed ' + (params.blp ? 'BLP ' : params.book ? 'BOOK' : '') + 'PROD. ~~~~~';
 			if (params.reason) {
-				logText += "\n#* '''Reason''': " + params.reason + '\n';
+				logText += "\n#* '''Reason''': " + params.reason.replace(/\n/g, '{{pb}}') + '\n';
```

Comment:
```suggestion
				logText += "\n#* '''Reason''': " + params.reason.replace(/\n+/g, '{{pb}}') + '\n';
```
Or `\n{2,}` to be more accurate and in line with normal wikitext representation.


para breaks in wikitext are `\n\n`.

Reply:
`\n{2,}` wouldn't work as we're talking about within a list, and even a single newline breaks out of that.

`+` is good!

I'm realizing now though that since successive para breaks don't replicate successive newlines — correct or not, hitting return 3+ times gives you more space — if we want truer-ish fidelity, we'd have to use {{break}}.

Reply:
For clarity, is that a +1 to `\n+` or `{{break}}`?

Reply:
Whichever you prefer. \n+ seems good enough. Haven't heard about {{break}}. I think it might be overkill.

Reply:
`\n+` done, as yes, I agree that {{break}} is a bit precious (and less easy to replicate elsewhere, although `br` would be simple I suppose).

I'm still debating whether it'd be better to put into `Morebits.string`, leaning yes since it's somewhat broadly useful, but then again still fairly limited in scope.

Reply:
> I'm still debating whether it'd be better to put into Morebits.string, leaning yes since it's somewhat broadly useful, but then again still fairly limited in scope.

Done in #1110, which added more of this sort of replacements.